.github/ISSUE_TEMPLATE/new_score.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: New Score or Metric
```

Comment:
Did you also want to update this line too?

Reply:
Can do. What about: "Request for a new metric, statistical test or tool"?